### DISCUSSION TITLE
osu!Lazer - Arch Linux VSync/Refresh rate

### DISCUSSION THREAD

**Discussion by MazOfficial** [ppy/osu#19871] [Q&A]
### Type

Performance

### Bug description

I downloaded .AppImage , the game works on my Arch 5.19.2 BUT , the VSync option set the fps to 60 but my monitor is 165hz
even if i set unlimited and have 300 fps , its still 60hz refresh rate, and i cant play smoothly. Any ideas?

### Screenshots or videos

![image](https://user-images.githubusercontent.com/90796065/185745726-08f1963e-7e5b-4c48-85b7-65d0f389e922.png)

### Version

2022.816.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9387078/database.log)
[network.log](https://github.com/ppy/osu/files/9387084/network.log)
[performance.log](https://github.com/ppy/osu/files/9387085/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9387086/runtime.log)

**Comment by mk56-spn** [comment]
This doesn't sound like it's OSU's fault because I run a very similar setup without issue . Assuming you're using KDE for arch do you have all the display settings properly configured

**Comment by MazOfficial** [comment]
I use xfce , its really annoying please bro i beg help me

W dniu sob., 20.08.2022 o 15:47 MK56 ***@***.***> napisał(a):

**Comment by Stoppedpuma** [comment]
Do you have another display plugged in that's 60hz? x11 locks the refresh rate to the lowest refresh rate display. It could also be your compositor which should suspend when you enable fullscreen exclusive (assuming osu! doesn't suspend it in all display modes on xfce).

**Comment by MazOfficial** [comment]
> Do you have another display plugged in that's 60hz? x11 locks the refresh rate to the lowest refresh rate display. It could also be your compositor which should suspend when you enable fullscreen exclusive (assuming osu! doesn't suspend it in all display modes on xfce).



That was the problem!

I close the topic. The issue was secound monitor, for some reason you cant have 165hz and 60hz at the same time on 2 monitors , the one will feel like 165 but the games wont give you more than 60hz. I fixed it by unplugging the 60hz monitor . Its possible to have 2 different hz but it needs a script to tell each monitor to give what hz.

**Reply by Stoppedpuma** [reply]
You could also consider switching to Wayland if you use vsync for most games, it will also fix the issue you're currently having.

**Reply by BlauFx** [reply]
If you can go with Wayland. However you can also disable the compositior.
